modules/programs/vdirsyncer.nix

```diff
@@ -24,8 +24,7 @@ let
 
   localStorage = a:
     filterAttrs (_: v: v != null)
-    ((getAttrs [ "type" "fileExt" "encoding" ] a.local) // {
-      path = a.local.path;
+    ((getAttrs [ "type" "fileExt" "encoding" "path" ] a.local) // {
```

Comment:
This seems unrelated to the urlCommand and userNameCommand support? If so then I would suggest putting this change in a separate PR.

Reply:
It is, just noticed that it was unnecessary to set it explicitly. I wouldn't bother to create a separate PR for this, so I guess it will remain as is forever :)